### DISCUSSION TITLE
Quick toggle between table / board

### DISCUSSION THREAD

**Discussion by chrysb** [community/community#12065] [Projects and Issues]
Excited by the promise of Github Projects!

One thing I noticed is I'd prefer to be able to quickly toggle between list and board layout within a View. Perhaps I can set the default, but otherwise I'd rather just toggle with one click vs switching the layout and having to "Save changes" if that makes sense.

**Comment by massung** [comment]
Because it's similar, I'll reference my suggestion with respect to this one: https://github.com/github/feedback/discussions/11971
